lib/tasks/deploy.rake

```diff
@@ -48,7 +48,14 @@ namespace :deploy do
     sh "docker build -t #{tag} ."
     sh "docker push #{tag}"
 
-    trigger_deployment(application, build, branch) if branch == 'staging'
+    next unless branch == 'staging'
+
+    deployed_apps = [application]
```

Comment:
Why is this in past tense? It's the list of applications to deploy to. How about `deploy_apps`, `target_apps`, or `deploy_targets`?